Corona-Warn-App/src/main/res/layout/include_test_result_positive_steps.xml

```diff
@@ -22,25 +22,13 @@
             app:step_entry_final="false"
             app:step_entry_icon="@drawable/ic_test_result_step_done" />
 
-        <de.rki.coronawarnapp.ui.view.SimpleStepEntry
-            android:id="@+id/test_result_positive_steps_positive_result"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/test_result_positive_steps_added"
-            app:simple_step_entry_text="@string/submission_test_result_positive_steps_positive_body"
```

Comment:
If this is the only usage if string ids, you could even delete them in the strings.xml files